views/posts.py

```diff
@@ -14,3 +16,26 @@ def list():
         posts = Post()
 
         return Response(json.dumps(posts.all(query, post_type), default=json_util.default), mimetype='application/json')
+
+
+class AddPostView(View):
+
+    @staticmethod
+    def perform_create():
+        req_data = request.get_json()
+
+        if req_data is not None:
```

Comment:
También podría ser
`if req_data is None:`
`    return Response(json.dumps({'error': 'Cannot create post. Post info is missing'}), status=400, mimetype='application/json')`

Y nos quitamos un nivel de indentación que nos gusta tan poco xD

Reply:
Efectivamente, toda la razón. De hecho lo voy a cambiar.